### PR TITLE
CSPM muted findings

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?

Documents the CSPM muted findings feature.

### Motivation

DOCS-4733

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/cspm/findings.md [hunk 1]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
 
 {{< img src="security/cspm/findings/passed-rules.png" alt="Group and aggregate by resource in search" style="width:100%;">}}
 
+## Mute findings
+
+{{< callout url="" btn_hidden="true" >}}
+  Muted findings is a beta feature available to all CSPM customers. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:67]:
```suggestion
  Muted findings is a beta feature available to all CSPM customers. If you have feedback or questions, contact <a href="/help">Datadog support</a>.
```

File: content/en/security/cspm/findings.md [hunk 2]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
...
+{{< /callout >}} 
+
+There may be times when a finding does not match the use case for your business, or you choose to accept it as a known risk. To ignore these findings, you can mute the finding for the impacted resource. This allows you to focus on high-severity and critical findings that require your attention.
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:70]:
```suggestion
Sometimes, a finding does not match the use case for your business, or you choose to accept it as a known risk. To ignore these findings, mute the finding for the impacted resource so you can focus on high-severity and critical findings.
```

File: content/en/security/cspm/findings.md [hunk 3]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
...
+
+For example, the ['Block Public Access' feature is enabled for S3 bucket][4] rule evaluates whether a S3 bucket is publicly accessible. If you have a S3 bucket with static assets that are meant to be publicly shared, you can mute the finding for the S3 bucket.
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:72]:
```suggestion
For example, the ['Block Public Access' feature is enabled for S3 bucket][4] rule evaluates whether an S3 bucket is publicly accessible. If you have an S3 bucket with static assets that are meant to be publicly shared, you can mute the finding for the S3 bucket.
```

File: content/en/security/cspm/findings.md [hunk 4]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
...
+
+You can mute pass/fail findings at any time. Muted findings do not impact your posture score.
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:74]: Does this mean the findings are still included when calculating the posture score? Or that they're omitted when calculating the score? I am guessing they're still included, in which case I would rephrase this as:
```suggestion
You can mute pass/fail findings at any time. Muting a finding does not remove it from the calculation of your posture score.
```

File: content/en/security/cspm/findings.md [hunk 5]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
...
+    - False positive
+    - Accepted risk
+    - Other
+4. Enter an optional **Description**.
+5. Select the duration of the mute:
+    - 2 weeks
+    - 1 month
+    - 3 months
+    - Indefinite
+    - Custom
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:91]: By listing these here, you're regurgitating what's already on the UI, and committing to keeping the list up to date. What if you said "Select a reason for the mute, for example, a fix is pending, it's a false positive, or it's an accepted risk." and for the duration, don't list them at all.

File: content/en/security/cspm/findings.md [hunk 6]
```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
...
+
+Muted findings automatically unmute after the specified mute duration expires. You can also manually unmute a finding.
+
+1. On the [finding side panel](#explore-your-cloud-misconfigurations-with-findings), select the resource with the muted finding.
+2. Click **Unmute**.
+3. Select a reason for the unmute:
+    - No Pending Fix
+    - Human Error
+    - No Longer Accepted Risk
+    - Other
```
Comment (kayayarai) [inline content/en/security/cspm/findings.md:104]: Same comment as above

**Review by kayayarai — CHANGES_REQUESTED** [review]
A few suggestions!

File: content/en/security/cspm/findings.md (hunk 5 above)
Comment (drichards-87) [reply content/en/security/cspm/findings.md:91]: Good point, and I like your suggestion. I'll update the PR with your recommendations.

File: content/en/security/cspm/findings.md (hunk 4 above)
Comment (drichards-87) [reply content/en/security/cspm/findings.md:74]: Actually, it's the former (muted findings are omitted from the calculation). I'll reword this sentence to make that distinction clearer to the reader.

**Review by kayayarai — APPROVED** [review]
